ha/setup/ha_setup.py

```diff
@@ -21,39 +21,21 @@
 import traceback
 import os
 import shutil
+import pathlib
 
 from cortx.utils.conf_store import Conf
 from cortx.utils.log import Log
 from cortx.utils.validator.v_pkg import PkgV
 from cortx.utils.security.cipher import Cipher
+sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..'))
```

Comment:
You kept it :). Should be ok I think.

Reply:
Yes i think i added it for testing but lets keep it as may need to handle if we remove pyinstaller
but due to pyinstaller this is getting deprecated and not affecting execution.

Reply:
Please remove this. When we remove pyinstaller, the package will go to standard path.

Reply:
ok, @ajay-paratmandali Please remove this.

Reply:
currently all entry point have this line and want to keep align with them

Reply:
currently if we run them from test/dev env we getting error ha module not found
as other way to add ha path to python path is not provided

Reply:
ok